prowler/providers/azure/services/storage/storage_service.py

```diff
@@ -42,11 +53,42 @@ def __get_storage_accounts__(self):
                 )
         return storage_accounts
 
+    def __get_blob_properties__(self):
```

Comment:
Can you add the test of this function in `storage_service_test.py`, pleae?

Reply:
Done!